README.md

```diff
@@ -56,6 +56,56 @@ module.exports = async x => x ** 2;
 module.exports = x => Promise.resolve(x ** 2);
 ```
 
+### Streams (experimental)
+
+Streaming functions can be created by setting the `$interactionModel` property on the function to `node-streams`.
+The function will then be invoked with two arguments, an `input` [Readable Stream](https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_class_stream_readable) and an `output` [Writeable Stream](https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_class_stream_writable).
+Both streams are object streams. Any value returned by the function is ignored, new messages must be written to the output stream.
+
+```js
+// echo.js
+module.exports = (input, output) => {
+    input.pipe(output);
+};
+module.exports.$interactionModel = 'node-streams';
+```
+
+Any npm package that works with Node Streams can be used.
+
+```js
+// upperCase.js
+const miss = require('mississippi');
+
+const upperCaser = miss.through.obj((chunk, enc, cb) => {
+    cb(null, chunk.toUpperCase());
+});
+
+module.exports = (input, output) => {
+    input.pipe(upperCaser).pipe(output);
+};
+module.exports.$interactionModel = 'node-streams';
+```
+
+The `Content-Type` for output messages can be set with the `$defaultContentType` property. By default, `text/plain` is used. For request-reply function, the `Accept` header is used, however, there is no Accept header in a stream.
```

Comment:
Was attempting to explain why there is a content-type concept exposed for streams, but not request-reply.

Reply:
The decorator is returning an object vs a string. Without the `$defaultContentType` property it would attempt to encode as text/plain and end up with messages like `[object Object]`. With `application/json` you get something like `{"original":"hello","upperCase":"HELLO"}`

Reply:
i still think one example with a note for the other case would be easier to grok, than two examples which are really similar.